app/build.gradle

```diff
@@ -148,7 +148,7 @@ android {
       debuggable true
       versionNameSuffix "-DEBUG"
       ext.enableCrashlytics = false
-      minifyEnabled true
+      minifyEnabled false
```

Comment:
is this change needed?

Reply:
Without this, I cannot step through the code.

Reply:
The buildApk task fails due to this, because it doesn't minify the debug build